eng/Version.Details.xml

```diff
@@ -5,12 +5,32 @@
       <Uri>https://github.com/dotnet/roslyn</Uri>
       <Sha>66a912c9463eebe832cf742d2fe8bb2e1a4600ec</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="8.0.0-alpha.1.23214.4">
+      <Uri>https://github.com/dotnet/source-build-reference-packages</Uri>
+      <Sha>05b4fdac346e01716abda4c92fcf0b7880e73792</Sha>
+      <SourceBuild RepoName="source-build-reference-packages" ManagedOnly="true" />
+    </Dependency>
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-externals" Version="8.0.0-alpha.1.23214.1">
+      <Uri>https://github.com/dotnet/source-build-externals</Uri>
+      <Sha>de4dda48d0cf31e13182bc24107b2246c61ed483</Sha>
+      <SourceBuild RepoName="source-build-externals" ManagedOnly="true" />
+    </Dependency>
   </ProductDependencies>
   <ToolsetDependencies>
     <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="8.0.0-beta.23213.1">
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>fb14cae30eacbe844468ac297b3d4c61e0bb9dc0</Sha>
       <SourceBuild RepoName="arcade" ManagedOnly="true" />
+    </Dependency>
+       <Dependency Name="Microsoft.SourceLink.GitHub" Version="8.0.0-beta.23211.2" CoherentParentDependency="Microsoft.DotNet.Arcade.Sdk">
+      <Uri>https://github.com/dotnet/sourcelink</Uri>
+      <Sha>759f344923a0859f3fae83431d0ba1cc62108118</Sha>
+      <SourceBuild RepoName="sourcelink" ManagedOnly="true" />
+    </Dependency>
+    <Dependency Name="Microsoft.DotNet.XliffTasks" Version="1.0.0-beta.23211.1" CoherentParentDependency="Microsoft.DotNet.Arcade.Sdk">
+      <Uri>https://github.com/dotnet/xliff-tasks</Uri>
+      <Sha>519d565b45c46ac452fe5a77ab63295138992e07</Sha>
+      <SourceBuild RepoName="xliff-tasks" ManagedOnly="true" />
     </Dependency>
     <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.5.0">
```

Comment:
This package is marked as pre-built and we cannot utilize a ref assembly for it. Bumping the package to the latest version causes a built-time exception:

```
/datadrive/source/roslyn-analyzers/artifacts/source-build/self/src/src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/TaintedDataAnalysis.TaintedDataOperationVisitor.cs(462,101): error CS8601: Possible null reference assignment. [/datadrive/source/roslyn-analyzers/artifacts/source-build/self/src/src/NetAnalyzers/Core/Microsoft.CodeAnalysis.NetAnalyzers.csproj]
```

Reply:
We'll need to fix the compile error. Because these 3 packages contain functionality that has to execute, them must either come from previously source-built, or from roslyn's build. Pinning to a stable version here works, but will probably mean more instability in product source build. Updating regularly from roslyn means more dependency flow and channel/subscription management.

I propose that we keep as-is for now, and if we see frequent breaks, we should move to change this to an actively updated dependency.